etc/profile-a-l/aria2p.profile

```diff
@@ -0,0 +1,65 @@
+# Firejail profile for aria2p
+# Description: Command-line tool and library to interact with an aria2c daemon process with JSON-RPC.
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include aria2p.local
+# Persistent global definitions
+include globals.local
+
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python3.inc
+
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+#include disable-write-mnt.inc
+# xclip and xsel require X11.
+#include disable-X11.inc
```

Comment:
@amano-kenji

Does the program itself use xclip/xsel directly?

Or do you just do something like `aria2p | xclip` on the CLI?

Reply:
> I don't really remember now, but aria2p itself doesn't use xclip or xsel
> directly.

(Continued at #6609)